### PR TITLE
[ADD] log_forwarded_for_ip: Displays source IPs in log when behind a …

### PR DESCRIPTION
**Pull request by pablocm-aserti**
…reverse proxy

### CODE REVIEW

File: log_forwarded_for_ip/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Forwarded-for IPs in log
+========================
+
+This module makes werkzeug messages in the log show the IP the requests come from instead of the reverse proxy one.
+
+
+Installation
+============
+
+Nothing additionally to installing the module.
```
Comment (elicoidal) [inline log_forwarded_for_ip/README.rst]: Only install the module in the database.

**Comment by dreispt** [comment]
Awesome!

**Comment by StefanRijnhart** [comment]
Excellent! :+1: code review and test.

**Comment by nhomar** [comment]
re-cool. :+1:

**Comment by ruiztulio** [comment]
@pablocm-aserti I like this (testing right now ) :+1

**Comment by pablocm-aserti** [comment]
Thank you! :)

**Pull request merged**